ehitchhiking-ui/src/app/shared/components/menu/menu.component.html

```diff
@@ -11,6 +11,9 @@
         <li *ngFor="let but of buttonsArray" class="li-buttons">
           <button mat-raised-button color="primary" class="li-button">{{ but }}</button>
         </li>
+        <button mat-stroked-button color="primary" class="log-out" (click)="openDialog()"><!--TODO: it's redundant here, but not now-->
```

Comment:
bad naming - openDialog. It is not clear what dialog opens

Reply:
fixed